Source/Classes/Categories/PINRemoteImageTask+Subclassing.h

```diff
@@ -0,0 +1,16 @@
+//
+//  PINRemoteImageTask+Subclassing.h
+//  PINRemoteImage
+//
+//  Created by Garrett Moon on 5/22/17.
+//  Copyright © 2017 Pinterest. All rights reserved.
+//
+
+#import "PINRemoteImageTask.h"
+
+@interface PINRemoteImageTask (Subclassing)
+
+- (nonnull NSMutableDictionary *)l_callbackBlocks;
```

Comment:
Might as well go in with `NS_ASSUME_NONNULL` here, and `nullable PINResume *_Nullable *` or whatever it is for the resume handler.